File: yetutil/src/main/java/net/yet/ui/activities/drawer/NavActionPanel.java
package net.yet.ui.activities.drawer;

import android.content.Context;
import android.widget.LinearLayout;

import net.yet.ui.util.XView;

/**
 * Created by entaoyang@163.com on 16/6/27.
 */
public class NavActionPanel extends LinearLayout {
	public NavActionPanel(Context context) {
		super(context);
		XView.view(this).orientationVertical();
	}
}

File: yetutil/src/main/java/yet/ui/activities/drawer/NavActionPanel.kt
package yet.ui.activities.drawer

import android.content.Context
import android.widget.LinearLayout
import yet.ui.ext.orientationVertical

/**
 * Created by entaoyang@163.com on 16/6/27.
 */
open class NavActionPanel(context: Context) : LinearLayout(context) {
	init {
		orientationVertical()
	}
}
